audbcards/core/templates/datacard_header.j2

```diff
@@ -0,0 +1,26 @@
+.. _{{ name }}:
+
+{{ name }}
+{% for n in range(name|length) %}-{% endfor %}
+
+
+{% if author is defined %}
+Created by {{ author }}
+{% endif %}
+
+============= ======================
+version       {{ version_link }}
+license       {{ license_link }}
+source        {{ source }}
+usage         {{ usage }}
+{% if languages is defined %}languages     {{ languages|join(', ') }} {% endif %}
```

Comment:
This fixes

> In content we get an extra space at the end of languages: \nlanguages     eng \n

from https://github.com/audeering/audbcards/pull/13#discussion_r1339665692.

```suggestion
{% if languages is defined %}languages     {{ languages|join(', ') }}{% endif %}
```

Reply:
Was updated by https://github.com/audeering/audbcards/pull/23